### PR TITLE
Drop Qt robot window

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Address #893:
* [x] Remove `gui_tracker` sample simulation
* ~~Remove `BotStudio` robot window~~
* [x] Remove OSM robot window
* [x] Remove automatic link to Qt libraries from `Makefile.include`
* [x] Update remaining Qt robot windows so that they still work:
    * [x] qtutils
    * [x] generic Qt robot window
    * [x] vehicle robot window
    * [x] darwin-op robot window
    * [x] fire bird 6 robot window
    * [x] BotStudio

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
:-1:

**Comment by stefaniapedrazzi** [comment]
:+1: :heavy_check_mark: (travis_unit)

**Comment by DavidMansolino** [comment]
:eyes:

File: .appveyor.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ install:
   - pacman --noconfirm -S mingw-w64-x86_64-cppcheck
   - pip install -r %APPVEYOR_BUILD_FOLDER%\\tests\\sources\\requirements.txt
   - pip install -r %APPVEYOR_BUILD_FOLDER%\\docs\\tests\\requirements.txt
-  - copy %APPVEYOR_BUILD_FOLDER%\\dependencies\\.clang-format-6 %APPVEYOR_BUILD_FOLDER%\\.clang-format
+  - copy %APPVEYOR_BUILD_FOLDER%\\dependencies\\.clang-format-9 %APPVEYOR_BUILD_FOLDER%\\.clang-format
```
Comment (DavidMansolino) [inline .appveyor.yml]: This is unrelated, but I assume it will disappear once the merge revision in develop is done.

Comment (stefaniapedrazzi) [reply .appveyor.yml]: Yes, it has already been merged to the master branch.

**Comment by stefaniapedrazzi** [comment]
I explicitly kept the sentence about BotStudio in `transfer-to-your-own-robot.md`.
From what I understand this is simply an example of what could be done.

**Comment by stefaniapedrazzi** [comment]
I tested locally on Linux and Windows and created a test Linux tarball package.

There was an issue downloading Python 3.8 for the travis sources test for Ubuntu 16.04 / Python 3.8 .
But the tests on the other travis setups passed.
I will try to run again at least the sources test.

:+1:

**Comment by stefaniapedrazzi** [comment]
The compilation on mac OS still failed while compiling the resources.
But at a different location than last time.

```
# *** resources ***
# downloading sumo-0.30-mac.tar.gz
# removing old ros
# extracting ros_headers.zip
make[3]: *** [sumo-0.30-mac.tar.gz] Error 16
make[2]: *** [resources.Makefile] Error 2
make[2]: *** Waiting for unfinished jobs....
# installing ros_headers.zip
make[1]: *** [release] Error 2
make: *** [webots_target] Error 2
The command "make distrib -j2" exited with 2.
$ if [[ "$(ls -l distribution | grep -v ^l | wc -l)" -ne "3" ]]; then echo Wrong number of files in the distribution folder; exit -1; fi
Wrong number of files in the distribution folder
```

Unfortunately without a mac OS system it is difficult for me to check if there is really a problem.

**Comment by DavidMansolino** [comment]
One of the nightly of this weekend had exactly the same problem on macOS, but the other worked fine, I am not sure to understand where does this come from.

**Comment by stefaniapedrazzi** [comment]
ok, thank you.
Besides the failure on mac OS, this PR is ready.
So if you agree I will raise the :+1:

**Comment by fabienrohrer** [comment]
👀

**Comment by stefaniapedrazzi** [comment]
@DavidMansolino do you want to review this PR?

**Comment by DavidMansolino** [comment]
:eyes:

**Comment by stefaniapedrazzi** [comment]
For code duplication, yes it would be good.
But then I'm not sure if a specific Makefile will encourage users to continue using Qt by providing easy tools to do it.

**Pull request merged**